operator.go

```diff
@@ -469,9 +469,17 @@ func New(opts ...Option) (*operator, error) {
 				if key != "" && key != k {
 					continue
 				}
-				val, err := newHttpValidator(&httpRunnerConfig{
+				runner, ok := bk.runners[k].(map[string]interface{})
```

Comment:
gostyle.ifacenames may be a false positive.

```suggestion
				runner, ok := bk.runners[k].(map[string]any)
```